code/game/objects/items/storage/storage.dm

```diff
@@ -445,6 +446,14 @@
 		I.mouse_opacity = initial(I.mouse_opacity)
 
 	update_icon()
+
+	if(isgun(I) && fumble_guns && user)
+		var/obj/item/weapon/gun/gun = I
```

Comment:
why casting? just use I.

Reply:
this is an artifact from when I was an idiot